client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -69,14 +79,40 @@ class PluginsBrowserListElement extends Component {
 		return ! this.props.isJetpackSite && includes( PREINSTALLED_PLUGINS, this.props.plugin.name );
 	}
 
+	isWpcomSupported() {
+		const { site, plugin } = this.props;
+
+		if ( ! site ) {
+			return false;
+		}
+
+		return (
+			( ! this.props.isJetpackSite && 'Automattic' === plugin.author_name ) ||
+			'WooCommerce' === plugin.author_name
+		);
+	}
+
+	renderSupportedFlag() {
+		if ( ! this.isWpcomSupported() ) {
+			return;
+		}
+
+		return (
+			<div className="plugins-browser-item__support">
+				{ this.props.translate( 'Support by' ) }
+				<img src={ logo } alt="WordPress.com" />
```

Comment:
Other languages may need to display the image in a different order. I guess it'd be good to inject the image into the translation: https://github.com/Automattic/wp-calypso/tree/trunk/packages/i18n-calypso#mixing-strings-and-markup

Reply:
TIL